test/sequential/test-regress-fs-watch_file.js

```diff
@@ -21,6 +21,17 @@
 
 'use strict';
 const common = require('../common');
+
+// Make sure the deletion event gets reported in the following scenario:
```

Comment:
`test-regress-fs-watch_file.js` seems a bit general. Maybe name this `test-fs-watchFile-enoent-after-deletion`?